src/licensedcode/data/licenses/ucar.LICENSE

```diff
@@ -0,0 +1,45 @@
+---
+key: ucar
+short_name: UCAR License
+name: UCAR License
+category: Permissive
```

Comment:
The starting matches to a bsd simplified detection, but there are terms and conditions about patents and counterclaims, so what should the category be?